serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

```diff
@@ -1 +1,164 @@
-= Orchestration of gRPC based services
\ No newline at end of file
+= Orchestrating the gRPC based services
+
+As a developer, you can invoke a gRPC service using Serverless Workflow application that is running with Quarkus. This guide describes how to create a workflow that invokes a running gRPC service and how to configure properties to locate a running gRPC service.
+
+For information about gRPC concepts, see link:https://grpc.io/docs/what-is-grpc/core-concepts/[gRPC].
+
+.Prerequisites
+* You have created a Serverless Workflow service. For information about creating your Serverless Workflow service, see xref:getting-started/create-your-first-workflow-service.adoc[Creating your first Serverless Workflow service] guide.
+
+In this guide, an example Serverless Workflow application is used that greets a user in their preferred language by invoking a gRPC service. The example Workflow application in this guide contains the following files:
+
+* link:https://github.com/kiegroup/kogito-examples/blob/main/kogito-quarkus-examples/serverless-workflow-greeting-rpc-quarkus/serverless-workflow-greeting-server-rpc-quarkus/src/main/proto/greeting.proto[gRPC proto file]: This file defines a greeter service, which consists of the `sayHello` method. The `sayHello` method accepts two parameters, including name of the user and an optional language to use in the greeting message (English by default). The `sayHello` method returns the greeting message in the appropriate language, such as:
```

Comment:
maybe for links, we can follow this best practice?
https://asciidoctor.org/docs/asciidoc-recommended-practices/#dry-urls